custom-configurations/src/main/java/com/sequenceiq/cloudbreak/domain/CustomConfigs.java

```diff
@@ -0,0 +1,140 @@
+package com.sequenceiq.cloudbreak.domain;
+
+import java.io.Serializable;
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+@Table(
+        uniqueConstraints = @UniqueConstraint(columnNames = {"name", "crn"})
+)
+public class CustomConfigs implements Serializable {
```

Comment:
should we rename this? just thinking loud here.

Reply:
Okay. Since this is the entity, it would require refactoring in all places, along with schema change.